faiss/utils/distances.cpp

```diff
@@ -146,7 +146,7 @@ void exhaustive_inner_product_seq(
 
     FAISS_ASSERT(use_sel == (sel != nullptr));
 
-#pragma omp parallel num_threads(nt)
+#pragma omp parallel num_threads(num_omp_threads)
```

Comment:
There is another function exhaustive_L2sqr_seq in this file with similar code. It would be good to change that as well while we are at it.

Reply:
Done.